napari/components/camera.py

```diff
@@ -28,9 +28,15 @@ class DepthAxisOrientation(StringEnum):
     TOWARDS = auto()
 
 
+class Handedness(StringEnum):
+    RIGHT = auto()
+    LEFT = auto()
+
+
 VerticalAxisOrientationStr = Literal['up', 'down']
 HorizontalAxisOrientationStr = Literal['left', 'right']
 DepthAxisOrientationStr = Literal['away', 'torwards']
+HandednessStr = Literal['right', 'left']
```

Comment:
I think this is not needed: it is not used anywhere. The orientation strings are there to type the *setter* for orientations, but we don't have a setter for handedness.

```suggestion
```